code/game/machinery/scanner_gate.dm

```diff
@@ -321,6 +364,11 @@
 						detect_nutrition = NUTRITION_LEVEL_FAT
 			. = TRUE
 
+/obj/machinery/scanner_gate/preset
```

Comment:
I don't really understand what "preset" means in this context. If its just guns, would be better named `scanner_gate/preset_guns` or something